mappings/net/minecraft/entity/vehicle/VehicleEntity.mapping

```diff
@@ -8,8 +8,8 @@ CLASS net/minecraft/unmapped/C_ogoqhclb net/minecraft/entity/vehicle/VehicleEnti
 	METHOD m_gbralwah isDestroyedBy (Lnet/minecraft/unmapped/C_sbxfkpyv;)Z
 		ARG 1 source
 	METHOD m_qobhgcrb killAndDropItem (Lnet/minecraft/unmapped/C_vorddnax;)V
-	METHOD m_tqtorwvl killAndDropSelf (Lnet/minecraft/unmapped/C_sbxfkpyv;)V
-		ARG 1 source
+	METHOD m_tqtorwvl onDamage (Lnet/minecraft/unmapped/C_sbxfkpyv;)V
```

Comment:
Considering the old name, is this method ran every time the vehicle is damaged?

Reply:
I accidentially mis-mapped this while mapping something else. Reverted to the old name as the new one wasn't correct.